gravitee-am-ui/src/app/domain/components/forms/form/dialog/form-info.component.html

```diff
@@ -35,6 +35,21 @@ <h1 matDialogTitle>Write your own {{data.template}} page</h1>
             <p th:if="${error}" class="error">
               <span>Wrong user or password</span>
             </p>
+ 	          <br>
+            <label>
+                <input type="checkbox" th:checked="${uc_geoip}" id="uc_geoip" name="uc_geoip">
+                <span id="#label-consent-ip">I consent to the collection and use of IP-location information for audit log purposes
+                    <span th:if="${userActivityEnabled}" th:text="${'and to improve security for a duration of ' + user_activity_retention_time}"></span>
+                </span>
+            </label>
+            <br>
+            <label>
+                <input type="checkbox" th:checked="${uc_ua}" id="uc_ua" name="uc_ua">
+                <span id="#label-consent-ua">I consent to the collection and use of User-Agent information for audit log purposes
+                    <span th:if="${userActivityEnabled}" th:text="${'and to improve security for a duration of ' + user_activity_retention_time}"></span>
+                </span>
+            </label>
+            <br>
```

Comment:
Maybe we should display a text message before the HTML snippet to explain that these two field are optional and useful only for RiskAssessment / Geo IP.

Reply:
FP